README.md

```diff
@@ -2,104 +2,116 @@
 
 [![test](https://github.com/mamba-org/provision-with-micromamba/workflows/test/badge.svg)](https://github.com/mamba-org/provision-with-micromamba/actions?query=workflow%3Atest)
 
-GitHub Action to provision a CI instance using micromamba
```

Comment:
You removed the inputs here, are the option descriptions rendered somewhere?

Reply:
Nope, do you want it back? I think it's ok to have people look at the `action.yml` file but I'm also OK with duplicating the docs.

Reply:
TBH I think it's nice to have the overview of supported settings right there in the readme. I didn't really know about the `action.yml` file...

Reply:
Ok, will bring it back.